helix-term/src/keymap.rs

```diff
@@ -600,4 +600,43 @@ mod tests {
             "Mismatch"
         )
     }
+
+    #[test]
+    fn escaped_keymap() {
+        use crate::commands::MappableCommand;
+        use helix_view::input::{KeyCode, KeyEvent, KeyModifiers};
+
+        let keys = r#"
+"+" = [
+    "select_all",
+    ":pipe sed -E 's/\\s+$//g'",
+]
+        "#;
+
+        let key = KeyEvent {
+            code: KeyCode::Char('+'),
+            modifiers: KeyModifiers::NONE,
+        };
+
+        let expectation = Keymap::new(KeyTrie::Node(KeyTrieNode::new(
+            "",
+            hashmap! {
+                key => KeyTrie::Sequence(vec!{
+                    MappableCommand::select_all,
+                    MappableCommand::Typable {
+                        name: "pipe".to_owned(),
+                        args: vec!{
```

Comment:
I don't like the number of `.to_owned()` here, but I also don't know of a good way to reduce them

Reply:
These are fine since you need a String. I changed it to `.to_string()` to match the test cases above though